Lib/unittest/test/testmock/testmock.py

```diff
@@ -2017,7 +2018,7 @@ def trace(frame, event, arg):  # pragma: no cover
             )
 
             mocks = [
-                Mock, MagicMock, NonCallableMock, NonCallableMagicMock
+                Mock, MagicMock, NonCallableMock, NonCallableMagicMock, AsyncMock
```

Comment:
I have been thinking to add this. Thanks for doing it. It also needs to be added for `EventMock` in the other PR just to make sure there is no regression.